lesson_11/test_log_event.py

```diff
@@ -0,0 +1,197 @@
+"""Test cases for login event logging function."""
+
+
+import logging
+import os
+
+import pytest
+
+from homework10 import log_event
+
+
+class TestLogEvent:
+    """
+    A class for testing the log_event function.
+
+    This class is responsible for logging events in the login system.
+    All tests verify different login statuses
+    (successful, failed, expired, etc.) and log the results into a file.
+
+    Input data for the tests:
+    - username: 'test_user#'
+    - status: 'success', 'failed', 'expired', or an invalid status
+
+    Expected result:
+    - Logs should contain correct messages with appropriate logging levels
+    (info, warning, error).
+    """
+
+    # First 4 TCs are similar, all about status
+    # Test Case N1
+    def test_log_event_success(*args):
+        """Checking the correct logging of the successful login event."""
+        # Clear log-file if exists in folder. Use os.path to find file
+        if os.path.exists(log_file):
+            with open(log_file, 'w'):
+                pass
```

Comment:
No need to do this each time. Better to do it once in tests setup or teardown . But we still had no lesson about fixtures - so just skip this stage . You could remove file manually now before all tests run. Or in tests just operate with last created line